packages/gatsby/src/cache-dir/static-entry.js

```diff
@@ -253,7 +276,7 @@ export default (pagePath, callback) => {
   )
 
   const bodyScripts = scripts.map(s => {
-    const scriptPath = `${pathPrefix}${JSON.stringify(s).slice(1, -1)}`
+    const scriptPath = `${pathPrefix}${JSON.stringify(s.name).slice(1, -1)}`
```

Comment:
We should add bundles that are marked to be prefetched here as that defeats the point of prefetching. Prefetching is lower priority in the background. If you add it as a script tag as well, it'll be loaded at high priority.

I'm  making another small fix in the codebase near your changes so will fix this too 👍)

Reply:
Thanks!